packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -132,7 +138,12 @@ export class BlockList extends Component {
 					extraScrollHeight={
 						blockToolbar.height + blockBorder.width
 					}
-					inputAccessoryViewHeight={ headerToolbar.height }
+					inputAccessoryViewHeight={
+						headerToolbar.height +
+						( isFloatingToolbarVisible
+							? floatingToolbar.height
+							: 0 )
+					}
```

Comment:
This will add additional space for FloatingToolbar when autoscroll to text base blocks